modules/serverpod_auth/serverpod_auth_server/lib/src/business/config.dart

```diff
@@ -47,6 +47,15 @@ typedef PasswordHashValidator = Future<bool> Function(
   void Function(Object e)? onError,
 });
 
+/// Enum to define the sign-out options for a user.
+enum SignOutOption {
```

Comment:
Suggestion/Discussion: Should this be renamed to `SignOutBehavior`?

Reply:
@SandPod  `LegacySignOutBehaviour` ??

Reply:
Thanks